### ISSUE TITLE
[Port] 1.20.1

### ISSUE THREAD

**Issue by thimblebird** [DiemondPlayer/IDiedMyXpDidnt#3]
would you consider supporting the same versions for both IDMID and IDMXD? landed here because i was wondering why i didn't install IDMXD (especially after reading their descriptions on modrinth), then saw there's no 1.20.1 version :(

**Comment by DiemondPlayer** [comment]
Hi! So, I actually planned for 1.20.x ports for IDMXD but unfortunately I found a bug that causes xp to turn invisible (their hitboxes too). You can pick up your xp but for some reason it just looks like it despawns when in reality it doesn’t. So, while the mod works on 1.20.x, because of this bug I didn’t release the 1.20.x versions yet. If I find how to fix the bug, or maybe I’ll just release it with this bug existing. Either way I can’t really do anything in the nearest week since I am away from my PC. I’ll reach out to you when I’ll release the 1.20.x versions

**Comment by DiemondPlayer** [comment]
TL;DR yes 1.20.x versions are planned but there is a bug that hasn’t been solved with those versions
